python/salpytools/salpylib.py

```diff
@@ -121,11 +123,11 @@ def subscribe_list(self,eventlist):
         for eventname in eventlist:
             self.subscribe_logEvent(eventname)
 
-    def send_logEvent(self,eventname,**kwargs):
+    def send_logEvent(self, eventname, **kwargs):
         ''' Send logevent for an eventname'''
         # Populate myData object for keys across logevent
-        kwargs.setdefault('timestamp',self.mgr[eventname].getCurrentTime())
-        kwargs.setdefault('priority',1)
+        kwargs.setdefault('timestamp', self.mgr[eventname].getCurrentTime())
+        kwargs.setdefault('priority', 1)
         priority = int(self.myData[eventname].priority)
```

Comment:
Why is the priority being captured before the kwarg has a chance to change it?

Reply:
Because for some unknown reason it cannot be changed to 1. It would be good to figured out why? Maybe a topic for a next EIA meeting?